### PR TITLE
Added new tools: prips, hakrevdns, nuclei & some commands in history

### PR DESCRIPTION
**Pull request by wlayzz**
Hey I added some new tools : 

- [Nuclei](https://github.com/projectdiscovery/nuclei)
- [Prips](http://www.ubuntugeek.com/prips-print-ip-address-on-a-given-range.html)
- [Hakrevdns](https://github.com/hakluke/hakrevdns)

I also add some example commands for these new tools & some useful commands history for pentest  engagement.

I have been working with exegol the past few days and it's really awesome ! Thx for the good work  :fire:

### CODE REVIEW

File: sources/zsh/history [hunk 1]
```diff
@@ -33,6 +33,8 @@ rpcclient -U BREAKING/anonymous 192.168.56.101
 pth-net rpc group members 'Domain admins' -U 'BREAKING.BAD'/'Administrator'%'ffffffffffffffffffffffffffffffff':'a88baa3fdc8f581ee0fb05d7054d43e4' -S 'DC01.BREAKING.BAD'
 pth-net rpc group addmem 'Domain admins' 'Shutdown' -U 'BREAKING.BAD'/'Administrator'%'ffffffffffffffffffffffffffffffff':'a88baa3fdc8f581ee0fb05d7054d43e4' -S 'DC01.BREAKING.BAD'
 cme smb 192.168.56.0/24 --gen-relay-list smb_targets.txt
+cme ldap DC01.BREAKING.BAD -d BREAKING.BAD -u someuser -p somepassword -M MAQ
```
Comment (ShutdownRepo) [inline sources/zsh/history:36]: If I remember correctly, MAQ should be written in lowercase, can you double check please?

File: sources/zsh/history [hunk 2]
```diff
@@ -33,6 +33,8 @@ rpcclient -U BREAKING/anonymous 192.168.56.101
...
+cme smb IP/CIDR --gen-relay-list smb_targets.txt
```
Comment (ShutdownRepo) [inline sources/zsh/history:37]: This history command already exists, please remove this duplicate

File: sources/zsh/history (hunk 1 above)
Comment (ShutdownRepo) [reply sources/zsh/history:36]: Nevermind, this is a duplicate to a command that's already written in the history, can you remove this line?

Comment (wlayzz) [reply sources/zsh/history]: Done

File: sources/zsh/history (hunk 2 above)
Comment (wlayzz) [reply sources/zsh/history]: Done

**Review by ShutdownRepo — APPROVED** [review]

**Pull request merged**